river/command/map.zig

```diff
@@ -30,6 +30,7 @@ const modifier_names = [_]struct {
     modifier: u32,
 }{
     .{ .name = "", .modifier = 0 },
```

Comment:
Sorry I didn't catch this in the first round, but we really shouldn't have two ways to do the same thing.
```suggestion
    .{ .name = "", .modifier = 0 },
```

Reply:
Agreed.